oracle/Dockerfile

```diff
@@ -6,7 +6,7 @@
 
 # Pull base image
 ARG BASE_IMAGE_TAG
-FROM openjdk:${BASE_IMAGE_TAG:-8u212-b04-jdk-stretch}
+FROM openjdk:${BASE_IMAGE_TAG:-11.0.2-jdk-oraclelinux7}
```

Comment:
Why default to 11 here and 8 above (debian)?

Reply:
This file with it's defaults was totally broken, using rpm on debian

- In the vars.sh there was only `12.0.1-jdk-oraclelinux7` which did not work for me so I brought it back to the same version as in use on the debian images (LTS jdk11)
- The lowest available openjdk jdk-oraclelinux7 seems to be 10.0.2 so 8 is not an option)
- There were several issues with newer versions so I settled for this one.